src/vivarium_testing_utils/automated_validation/data_transformation/formatting.py

```diff
@@ -40,11 +40,33 @@ class TransitionCounts(SimDataFormatter):
     """Formatter for simulation data that contains transition counts."""
 
     def __init__(self, cause: str, start_state: str, end_state: str) -> None:
-        super().__init__("transition_count", cause, f"{start_state}_to_{end_state}")
+        super().__init__(
+            measure="transition_count",
+            entity=cause,
+            filter_value=f"{start_state}_to_{end_state}",
+        )
 
 
 class PersonTime(SimDataFormatter):
     """Formatter for simulation data that contains person time."""
 
     def __init__(self, cause: str, state: str | None = None) -> None:
```

Comment:
Perhaps this should be called `StatePersonTime`? I also think `cause` should be renamed to `model`, since that's more generic.